frontend/public/module/k8s/selector-requirement.js

```diff
@@ -8,42 +8,35 @@ export const createEquals = (key, value) => ({
   values: [value],
 });
 
+const validateKey = (key, operator, values) => {
+  if (/^(([A-Za-z0-9][-A-Za-z0-9.]*)?[A-Za-z0-9])$/.test(key)) {
```

Comment:
Yeah, that was not right. I switched it to:
```
const validateKey = (key) => /^(([A-Za-z0-9][0-9A-Za-z/\-_.]*)?[A-Za-z0-9])$/.test(key);
```
Testing looks correct.

Reply:
That's still not accurate, though. There are two parts: an optional prefix and a name. They have different rules.

Should we consider just fixing the runtime error when there are no results and showing the error message from the server when the selector is bad? I feel like we're duplicating too much server logic here to be honest.

Reply:
The runtime error appears in the virtualized vendor code.  Can you direct me on what I should look at? 

I am okay not checking this on the console side only because we are replicating the work.  That is, only if we can figure out how to prevent the empty table from exploding.

Reply:
I guess I'm unclear why the virtualization code is running at all. If there's an error, we shouldn't have a table to show. Is the table still being rendered?